### ISSUE TITLE
scilab_kernel exiting with file not found error

### ISSUE THREAD

**Issue by Madouc** [Calysto/scilab_kernel#3]
I have installed anaconda with jupyter,
then installed the scilab_kernel as specified in the readme, but when i try to open the example ipynb the kernel fails with the following trace:

```
[I 18:57:48.669 NotebookApp] KernelRestarter: restarting kernel (4/5)
WARNING:root:kernel 691952f0-c357-487d-bd8b-eddc4a05a04b restarted
Traceback (most recent call last):
  File "C:\Users\BWithagen\AppData\Local\Continuum\Anaconda3\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\Users\BWithagen\AppData\Local\Continuum\Anaconda3\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\BWithagen\AppData\Local\Continuum\Anaconda3\lib\site-packages\scilab_kernel\__main__.py", line 6, in <module>
    IPKernelApp.launch_instance(kernel_class=ScilabKernel)
  File "C:\Users\BWithagen\AppData\Local\Continuum\Anaconda3\lib\site-packages\traitlets\config\application.py", line 657, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-124>", line 2, in initialize
  File "C:\Users\BWithagen\AppData\Local\Continuum\Anaconda3\lib\site-packages\traitlets\config\application.py", line 87, in catch_config_error
    return method(app, *args, **kwargs)
  File "C:\Users\BWithagen\AppData\Local\Continuum\Anaconda3\lib\site-packages\ipykernel\kernelapp.py", line 457, in initialize
    self.init_kernel()
  File "C:\Users\BWithagen\AppData\Local\Continuum\Anaconda3\lib\site-packages\ipykernel\kernelapp.py", line 368, in init_kernel
    user_ns=self.user_ns,
  File "C:\Users\BWithagen\AppData\Local\Continuum\Anaconda3\lib\site-packages\traitlets\config\configurable.py", line 412, in instance
    inst = cls(*args, **kwargs)
  File "C:\Users\BWithagen\AppData\Local\Continuum\Anaconda3\lib\site-packages\metakernel\process_metakernel.py", line 54, in __init__
    self.wrapper = self.makeWrapper()
  File "C:\Users\BWithagen\AppData\Local\Continuum\Anaconda3\lib\site-packages\scilab_kernel\kernel.py", line 81, in makeWrapper
    prompt_emit_cmd=prompt_cmd)
  File "C:\Users\BWithagen\AppData\Local\Continuum\Anaconda3\lib\site-packages\metakernel\replwrap.py", line 65, in __init__
    encoding="utf-8")
  File "C:\Users\BWithagen\AppData\Local\Continuum\Anaconda3\lib\site-packages\metakernel\pexpect.py", line 31, in spawn
    encoding=encoding, codec_errors=codec_errors)
  File "C:\Users\BWithagen\AppData\Local\Continuum\Anaconda3\lib\site-packages\pexpect\popen_spawn.py", line 45, in __init__
    self.proc = subprocess.Popen(cmd, **kwargs)
  File "C:\Users\BWithagen\AppData\Local\Continuum\Anaconda3\lib\subprocess.py", line 707, in __init__
    restore_signals, start_new_session)
  File "C:\Users\BWithagen\AppData\Local\Continuum\Anaconda3\lib\subprocess.py", line 990, in _execute_child
    startupinfo)
FileNotFoundError: [WinError 2] The system cannot find the file specified
[W 18:57:51.682 NotebookApp] KernelRestarter: restart failed
```

**Comment by blink1073** [comment]
Hi @Madouc, it looks like the Scilab executable directory is not in your PATH environment variable.

**Comment by Madouc** [comment]
i have placed both
SCILAB_EXECUTABLE=C:\Program Files\scilab-6.0.0\bin
and the same directory on the path, but still get the same error.

**Comment by blink1073** [comment]
Does it work if you use the full name (including the *.exe) in SCILAB_EXECUTABLE?

**Comment by blink1073** [comment]
Closing for lack of response.

**Comment by pedrohnv** [comment]
I just installed `scilab_kernel` with `pip` and I am having this issue. I've put the full name (including the *.exe) in both SCILAB_EXECUTABLE and PATH enviromental variables and the same error happens.

I've made sure Python can see the variable (it does) with `os.getenv("SCILAB_EXECUTABLE")`. My scilab (64 bit) is installed in `C:\scilab-6.0.0\bin\WScilex.exe`, Windows 10 (64 bit).

**Comment by pedrohnv** [comment]
I just realized that my SCILAB_EXECUTABLE enviroment variable was pointing to `WScilex.exe`, which is the GUI console of Scilab. The NO GUI version is `Scilex`. And you have changed in that [commit](https://github.com/Calysto/scilab_kernel/commit/eb4d7317d8414a21f828f6e4776aeb3c27e0f4ca) the `executable` in line 62 in `kernel.py` from `Scilex` to `WScilex`. I think that may lead to further issues. Sorry for my mistake.

Anyway: I've corrected my SCILAB_EXECUTABLE variable, reverted `scilab_kernel` to `0.9.0`. The issue still happens in that version (but is corrected in `0.9.1`).
